.github/workflows/test_suite.yml

```diff
@@ -39,10 +39,23 @@ jobs:
         shell: bash
         run: ${SCCACHE_PATH} --show-stats
 
-      - name: Install rdkit
+      - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y build-essential librdkit-dev libssl-dev libboost1.74-dev libboost-serialization1.74-dev pkg-config
+          sudo apt-get install -y build-essential libssl-dev libboost-iostreams-dev libboost-serialization-dev pkg-config curl tar
+
+      - name: Install RDKit
+        run: |
+          cd /tmp
+          if [ $(dpkg --print-architecture) = "amd64" ]; then
```

Comment:
bash `if` statements are tricky. can you try `if [[ $(...) == "amd64"]]; then`

Reply:
Weirdly, when I made this change, the build failed. The error complained about the *.so files being incompatible, which makes me thing it started getting the wrong tar ball.